bCNC/__main__.py

```diff
@@ -1,39 +1,10 @@
-#!/usr/bin/python
-# $Id: bCNC.py,v 1.6 2014/10/15 15:04:48 bnv Exp bnv $
-#
-# Author: vvlachoudis@gmail.com
-# Date: 24-Aug-2014
+#!/usr/bin/env python3
 
-from __future__ import absolute_import, print_function
-
-import getopt
 import os
-import socket
 import sys
-import threading
-import time
-import traceback
-import webbrowser
-from datetime import datetime
+import getopt
 
-try:
-    import serial
-except ImportError:
-    serial = None
-    print("testing mode, could not import serial")
-try:
-    import Tkinter
-    from Queue import *
-    from Tkinter import *
-    import tkMessageBox
-except ImportError:
-    import tkinter
-    from queue import *
-    from tkinter import *
-    import tkinter.messagebox as tkMessageBox
-except ImportError:
-    print("tkiner is not installed on this machine")
-    exit(1)
+import Helpers # required for translation
```

Comment:
@m1ch Testing this on python 3.10, windows 10. results in:
```
D:\Users\bosd>py -m bCNC
Traceback (most recent call last):
  File "D:\Users\bosd\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "D:\Users\bosd\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "D:\Users\bosd\AppData\Local\Programs\Python\Python310\lib\site-packages\bCNC\__main__.py", line 7, in <module>
    import Helpers # required for translation
ModuleNotFoundError: No module named 'Helpers'
```


Tried changing it to:
`from . import Helper`

results in main window not loading, traceback.

```
Traceback (most recent call last):
  File "D:\Users\bosd\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "D:\Users\bosd\AppData\Local\Programs\Python\Python310\lib\runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "D:\Users\bosd\AppData\Local\Programs\Python\Python310\lib\site-packages\bCNC\__main__.py", line 228, in <module>
    main()
  File "D:\Users\bosd\AppData\Local\Programs\Python\Python310\lib\site-packages\bCNC\__main__.py", line 48, in main
    import bmain
  File "D:\Users\bosd\AppData\Local\Programs\Python\Python310\lib\site-packages\bCNC\bmain.py", line 83, in <module>
    Utils.loadConfiguration()
  File "D:\Users\bosd\AppData\Local\Programs\Python\Python310\lib\site-packages\bCNC\Utils.py", line 218, in loadConfiguration
    lang = gettext.translation(
  File "D:\Users\bosd\AppData\Local\Programs\Python\Python310\lib\gettext.py", line 592, in translation
    raise FileNotFoundError(ENOENT,
FileNotFoundError: [Errno 2] No translation file found for domain: 'bCNC'
```

Reply:
I tested it now in Windows and applied the required fixes